sdk/core/AzureCore/Source/Util/RequestString.swift

```diff
@@ -115,6 +115,18 @@ extension Array: RequestStringConvertible {
     }
 }
 
+extension Dictionary: RequestStringConvertible {
+    public var requestString: String {
+        do {
+            let jsonData = try JSONSerialization.data(withJSONObject: self)
```

Comment:
None of the other RequestStringConvertable implementations serialize to JSON, is this appropriate?

Reply:
Yes. This is what Autorest.Swift has to do to serialize dictionaries. The implementation here simply moves the complexity out of autorest.